packages/taucmdr/cf/software/installation.py

```diff
@@ -225,6 +225,8 @@ def __init__(self, name, title, sources, target_arch, target_os, compilers,
                 self.srcs_avail = [self.src]
         elif src.lower() == 'download-tr6':
             raise ConfigurationError("download-tr6 is not a valid source for %s" % self.title)
+        elif src.lower() == 'download-tr4':
+            raise ConfigurationError("download-tr4 is not a valid source for %s" % self.title)
```

Comment:
I thought we dropped tr4 support since we have tr6. But I may be mistaken, please double check that this is correct.